blade-graphics/src/gles/resource.rs

```diff
@@ -159,7 +172,11 @@ impl crate::traits::ResourceDevice for super::Context {
                     if desc.array_layer_count > 1 {
                         glow::TEXTURE_2D_ARRAY
```

Comment:
can we support this too? or leaving for a follow-up?
we don't have to do validation in Blade, but it would be polite to at least do a warning
```
log::warn!("Sample count is ignored: not supported for this texture type yet");
```

Reply:
Was unsure what to do there, since in that case maybe we should warn about the D3 and D1 case as well, and then about the mip-levels being ignored too. I agree some nice-to-have warnings are appreciated as the user xD Where to stop~